src/Symfony/Component/Config/Resource/FileResource.php

```diff
@@ -44,7 +44,7 @@ public function __toString()
     }
 
     /**
-     * {@inheritdoc}
+     * @return string The file path to the resource
```

Comment:
Hm, maybe. But that has nothing to do with this PR, or? Shouldn't that be a differnt PR possibly with a test?